go-controller/pkg/ovn/namespace.go

```diff
@@ -61,8 +63,7 @@ func (oc *Controller) getRoutingExternalGWs(nsInfo *namespaceInfo) *gatewayInfo
 	// return a copy of the object so it can be handled without the
 	// namespace locked
 	res.bfdEnabled = nsInfo.routingExternalGWs.bfdEnabled
-	res.gws = make([]net.IP, len(nsInfo.routingExternalGWs.gws))
-	copy(res.gws, nsInfo.routingExternalGWs.gws)
+	res.gws = nsInfo.routingExternalGWs.gws
```

Comment:
@tssurya won't this pass back the same sets.String struct/object and open up to concurrency issues? I think that's what the copy() was for IIRC.

Should we do `res.gws = nsInfo.routingExternalGWs.gws.Difference(NewString())` or something to return a new set with the same data?

Reply:
`sets.NewString(nsInfo.routingExternalGWs.gws.UnsortedList()...)` went with that, since it seems to give a new set with the given items: https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#NewString